src/Service/ZGWToXxllncService.php

```diff
@@ -376,6 +530,60 @@ public function syncZaakToXxllnc(): array
     }//end syncZaakToXxllnc()
 
 
+    /**
+     * Handles all code to send a zgw zaak as a xxllnc case to the xxllnc v1 api.
+     *
+     * @param string|null $zaakTypeId The id of the zaaktype
+     *
+     * @return array empty
+     * @throws Exception
+     */
+    public function syncZaakTypeToXxllnc(): array
```

Comment:
You could combine some of the if isset checks here into less if statements (optional)